auction-app-frontend/src/components/ProductBidsTable/ProductBidsTable.jsx

```diff
@@ -16,6 +16,32 @@ const ProductBidsTable = ({ emptyMessageComponent, items, buttonLabel, tabId })
     navigate(`${ ROUTE_PATHS.PRODUCT }/${ id }`);
   }
 
+  const adjustItemData = (items) => {
+    return items.map(item => {
+      // check if nested
+      if (item.product && item.product.id) {
```

Comment:
Can you explain this, why is this happening?

Reply:
because seller tab and bids tab are sending different structured items, so here i'm just organizing them better so i can reuse the product bids table and bids card